etc-conf/zypper.conf

```diff
@@ -3,7 +3,10 @@
 enabled = 1
 
 # When gpgcheck is set to 0, then 'gpgcheck' will be set to 0 in generated repo file too.
-gpgcheck = 0
+gpgcheck = 1
```

Comment:
Is it necessary to change default behavior?

Reply:
I'm not sure. Checking gpg keys by default isn't the worst idea, is it?

Reply:
Maybe it is not the worst idea, but I'm almost sure that it isn't the best idea, because when you change default behavior, then it has potential to put somebody into the trouble... What has worked in the past just stopped to work after upgrade of software.

You can set `gpgcheck` to True in definition of repository in entitlement certificate. You can set gpgcheck to True in zypper.conf. If some users require checking of GPG, then they already have it.

So, I would not change default behavior.

Reply:
I reverted the default behaviour and changed the comment above the check as you requested